docs/Converters/ADNI2BIDS.md

```diff
@@ -100,6 +102,11 @@ The user can easily modify this file if they want to convert additional clinical
 For further details regarding clinica data, we recommend to look at the [ADNI Data Package
 ](https://adni.bitbucket.io/index.html) developed by the Alzheimer's Disease Neuroimaging Initiative.
 
+??? failure "Warning : Field Mapping modality"
```

Comment:
I don't see this block in the rendered documentation: https://aramislab.paris.inria.fr/clinica/docs/public/PR-1199/Converters/ADNI2BIDS/

Reply:
I don't see it either, as well as all the changes I made in the last commit (ex : removing the "except for FMAP" part as suggested). Did I miss something I should have done for it to appear ?

Reply:
Indeed, there is a bug in the automatic documentation deployment pipeline... 😱 I'll open a PR to fix it.
I've updated the docs for this PR manually and the changes you made seem to be there.
Could you just double check that everything looks good in the rendered page ?

Reply:
I am not sure if it is still relevant now that you merged the PR but no I can't see the new changes in the rendered page. I will rebase a bit later :-)

Reply:
That's weird, I can see them.